src/Sulu/Bundle/SecurityBundle/Controller/ResettingController.php

```diff
@@ -224,11 +236,25 @@ public function sendEmailAction(Request $request)
         try {
             /** @var UserInterface $user */
             $user = $this->findUser($request->get('user'));
+        } catch (EntityNotFoundException $ex) {
+            // do not report when there is no user, because this could be used for username enumeration attacks
+            return new JsonResponse(null, Response::HTTP_NO_CONTENT);
+        } catch (UserNotInSystemException $ex) {
+            if ($this->logger instanceof LoggerInterface) {
+                $this->logger->error($ex->getMessage(), ['exception' => $ex]);
+            }
+            $response = new JsonResponse(null, Response::HTTP_BAD_REQUEST);
+        }
+
+        try {
             $token = $this->generateTokenForUser($user);
             $email = $this->getEmail($user);
             $this->sendTokenEmail($user, $this->getSenderAddress($request), $email, $token);
         } catch (\Exception $ex) {
-            // do nothing
+            if ($this->logger instanceof LoggerInterface) {
+                $this->logger->error($ex->getMessage(), ['exception' => $ex]);
```

Comment:
Errors should only be logged for unexpected exceptions. here a lot of expected exception happens. So either change the logger to debug.

```suggestion
                $this->logger->debug($ex->getMessage(), ['exception' => $ex]);
```